components/Article/Article.module.scss

```diff
@@ -14,13 +14,22 @@
   display: -webkit-box;
   -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
-  overflow: hidden;
+
+  
   *:not(code) {
     background-color: transparent!important;
   }
   @include screenMax576 {
     font-size: 1rem;
   }
+  code{
```

Comment:
format this block with Prettier, it is very ugly

Reply:
This bug was fixed